rust/processor/src/db/common/models/token_v2_models/parquet_v2_collections.rs

```diff
@@ -0,0 +1,306 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::{
+        object_models::v2_object_utils::ObjectAggregatedDataMapping,
+        token_models::{
+            collection_datas::CollectionData,
+            token_utils::{CollectionDataIdType, TokenWriteSet},
+            tokens::TableHandleToOwner,
+        },
+        token_v2_models::{
+            v2_collections::CreatorFromCollectionTableV1,
+            v2_token_utils::{TokenStandard, V2TokenResource},
+        },
+    },
+    utils::{database::DbPoolConnection, util::standardize_address},
+};
+use allocative_derive::Allocative;
+use anyhow::Context;
+use aptos_protos::transaction::v1::{WriteResource, WriteTableItem};
+use bigdecimal::{BigDecimal, Zero};
+use diesel::{sql_query, sql_types::Text};
+use diesel_async::RunQueryDsl;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct CollectionV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub collection_id: String,
+    pub creator_address: String,
+    pub collection_name: String,
+    pub description: String,
+    pub uri: String,
+    pub current_supply: String,
+    pub max_supply: Option<String>,
+    pub total_minted_v2: Option<String>,
+    pub mutable_description: Option<bool>,
+    pub mutable_uri: Option<bool>,
+    pub table_handle_v1: Option<String>,
+    pub token_standard: String,
+    #[allocative(skip)]
```

Comment:
we used to also have a `collection_properties` field

Reply:
collections is not going to be migrated over in this phase. we can ignore this file for now